src/hotspot/share/opto/node.hpp

```diff
@@ -335,8 +335,11 @@ class Node {
   void grow( uint len );
   // Grow the output array to the next larger power-of-2 bigger than len.
   void out_grow( uint len );
+  // Resize input or output array to grow it the next larger power-of-2 bigger
+  // than len.
+  void resize_array(Node **&array, node_idx_t &max_size, uint len, bool is_in);
```

Comment:
```suggestion
  void resize_array(Node**& array, node_idx_t &max_size, uint len, bool is_in);
```
Nit code style ;)

Reply:
What does `is_in` mean here? Feel free to give it a longer more expressive name ;)